x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild

```diff
@@ -34,6 +31,7 @@ src_prepare() {
 		cp "${x}"/places/scalable/start-here-gentoo.svg \
 			"${x}"/places/scalable/start-here.svg || die
 	done
+	default
```

Comment:
We inherit it, so let's run `xdg_src_prepare` here.

Reply:
done